src/app/store.ts

```diff
@@ -1,13 +1,19 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
 import counterReducer from '../features/counter/counterSlice';
+import usersReducer, { fetchUsers } from '../features/users/usersSlice';
+import postsReducer from '../features/posts/postSlice';
```

Comment:
Ensure that the `postsReducer` is correctly implemented and exported in the `../features/posts/postSlice` file. Verify its existence and functionality.